core/src/main/java/hudson/XmlFile.java

```diff
@@ -352,13 +351,14 @@ private void attempt() throws Eureka {
     /**
      * {@link XStream} instance is supposed to be thread-safe.
      */
-    private static final XStream DEFAULT_XSTREAM = new XStream2();
 
     private static final Logger LOGGER = Logger.getLogger(XmlFile.class.getName());
 
     private static final SAXParserFactory JAXP = SAXParserFactory.newInstance();
 
-    private static final Xpp3Driver DEFAULT_DRIVER = new Xpp3Driver();
+    private static final HierarchicalStreamDriver DEFAULT_DRIVER = XStream2.getDefaultDriver();
```

Comment:
Do we not want to be specific about the default driver being `kXML`?  in case a future xStream changes it and causes some issues?  (I guess tests would pick this up?)